application/models/Template.php

```diff
@@ -147,7 +147,7 @@ public static function templateNameFilter($sTemplateName)
         /* Validate if template is OK in user dir, DIRECTORY_SEPARATOR not needed "/" is OK */
         $oTemplate = self::model()->findByPk($sTemplateName);
 
-        if (is_object($oTemplate) && $oTemplate->checkTemplate() && (self::checkTemplateXML($oTemplate->folder))) {
+        if (is_object($oTemplate) && $oTemplate->checkTemplate() && $oTemplate->name === $sTemplateName && self::checkTemplateXML($oTemplate->folder)) {
```

Comment:
This condition here ` $oTemplate->name === $sTemplateName` resembles a lot of a filter.
Have you thought about overriding findByPk on the template class?
In there, after finding we would apply this extra filter. Return null if doesn't full fills.

Maybe too wide of an update?

Reply:
> Have you thought about overriding findByPk on the template class?
No, it's a great idea !

> Maybe too wide of an update?

I don't think so, i create an update of this fix ? Your opinion ?

Reply:
I don't understand the whole semantics of this change, but seems templatename case spelling is a central rule, so why not bringing it into the Model?

Initially in the findBYPk, but later on some validations...

> I don't think so, i create an update of this fix ? Your opinion ?

This is already merged, so maybe add it like a DEV PR?
I would say to go for it. This is one of those rules that can easily layer get lost in code but seems important...

Reply:
> I don't understand the whole semantics of this change, but seems templatename case spelling is a central rule, so why not bringing it into the Model?

I already answer it's a good idea :)

> This is already merged, so maybe add it like a DEV PR?

OK i create a DEV mantis issue